src/client/pages/Vergunningen/Vergunningen.tsx

```diff
@@ -66,7 +67,20 @@ export default function Vergunningen() {
   }, [VERGUNNINGEN.content]);
 
   const vergunningenPrevious = useMemo(() => {
-    return vergunningen.filter((vergunning) => vergunning.processed);
+    return vergunningen
+      .filter((vergunning) => vergunning.processed)
+      .map((vergunning) => {
+        if (
+          vergunning.caseType === CaseType.WVOS &&
+          hasMultiplePermits(vergunning)
+        ) {
+          return {
+            ...vergunning,
+            decision: 'Zie besluit',
+          };
+        }
+        return vergunning;
+      });
```

Comment:
Kun je deze logica niet beter onderbrengen in iets simpels in de WVOS.tsx pagina? je gaat hier nu WVOS specifieke checks langs alle vergunningen halen...

**Pseudocode:**
```
<span>{hasMultiplePermits(vergunning) ? 'Zie besluit' : vergunning.decision}</span>
```

Reply:
ja maar dit wil je zien op de list page van vergunningen

Reply:
Ok duidelijk. Wordt dat resultaat ook op een detailpagina getoond? Dan moet dit daar ook nog neergezet worden.